src/printer_tree.c

```diff
@@ -97,6 +85,12 @@ struct ly_out_clb_arg {
     LY_ERR last_error;                  /**< The last error that occurred. If no error has occurred, it will be LY_SUCCESS. */
 };
 
+/**
+ * @brief Initialize struct ly_out_clb_arg with default settings.
+ */
+#define TRP_DEFAULT_LY_OUT_CLB_ARG \
+    (struct ly_out_clb_arg){.mode = TRD_PRINT, .out = out, .counter = 0, .last_error = LY_SUCCESS}
```

Comment:
When the macro is not defined just temporarily inside a function, do not use variables directly, define that variable `OUT` for the macro and use it in initialization)